cmake/compilers/GNU.cmake

```diff
@@ -56,7 +56,7 @@ endif()
 
 # Workaround for heavy tests and too many symbols in debug (rellocation truncated to fit: R_MIPS_CALL16)
 if ("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "mips")
-    set(TBB_TEST_COMPILE_FLAGS ${TBB_TEST_COMPILE_FLAGS} -DTBB_TEST_LOW_WORKLOAD $<$<CONFIG:DEBUG>:-mxgot>)
+    set(TBB_TEST_COMPILE_FLAGS ${TBB_TEST_COMPILE_FLAGS} -DTBB_TEST_LOW_WORKLOAD $<$<CONFIG:DEBUG>:-fPIE -mxgot>)
```

Comment:
Consider setting `-fPIE` in `cmake/toolchains/mips.cmake` as you do for linker flag

Reply:
@AlexVeprev As I understand, there is no such variable as `CMAKE_EXE_CXX_FLAGS`. We need to set `-fPIE` together with `-mxgot` only for tests only in debug. We can leave this as is or move this whole line to `cmake/toolchains/mips.cmake`.